content/docs/use-cases/data-registries.md

```diff
@@ -16,28 +16,26 @@ repository would have all the metadata and history of changes in the different
 datasets. We could see who updated what and when, and use pull requests to
 update data, the same way we do with code.
 
-This is what we call a **data registry** — a kind of data management
-_middleware_ between ML projects and cloud storage. Here are its advantages:
+That's a **data registry**: a data management _middleware_ between ML projects
```

Comment:
not sure - why do we touch the second use case here?

Reply:
I initially reused some of the advantages from the bullet list in this doc. In the end we only kept one fo them as overlap, I think, but still I worked on the 2 docs in parallel a little bit. This particular line is an unrelated copy edit I noticed and decided to throw in.